tools/p11ne-cli

```diff
@@ -158,11 +158,18 @@ ensure_nitro_cli() {
 #
 ensure_aws_creds() {
     local role
-    role=$(curl -fs http://169.254.169.254/latest/meta-data/iam/security-credentials/)
+    TOKEN=`curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600"`
+    ok_or_die "Error: p11ne could not fetch IMDSv2 session token"
+    [[ -z $TOKEN ]] && die "Error: invalid IMDSv2 session token"
+
+    curl -H "X-aws-ec2-metadata-token: $TOKEN" -v http://169.254.169.254/latest/meta-data/
+    ok_or_die "Error: p11ne could not fetch the IMDS medatada"
```

Comment:
Should be `metadata`

Reply:
In the URL? Its `meta-data`. 

Consistent with the guide: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html

Also, this was tested and worked.